template/Goerli-DssSpellCollateral.sol

```diff
@@ -0,0 +1,168 @@
+// SPDX-FileCopyrightText: © 2021-2022 Dai Foundation <www.daifoundation.org>
```

Comment:
The best would be to actuall `git mv` the old file to the new name, so that way github still recognizes the history.

Reply:
I used `git mv` and committed both `git mv` and updates in the same commit, as mentioned in chat.
I've now undo those changes and repeat the steps separately but It looks the same, where the old file is deleted instead of renamed.